### PR TITLE
allow use of custom TeX Live repository

### PR DESCRIPTION
**Pull request by anneFly**
Makes the texlive repository configurable in case you need to install older versions of texlive.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,21 +45,26 @@ How does it work?
...
     collection-langgerman
     collection-xetex
 
-It looks similar to the default `texlive.profile`, but without the `1` or `0` at the end. The buildpack runs `tlmgr install` on every line in this file. So you can use single packages or these collections. 
+It looks similar to the default `texlive.profile`, but without the `1` or `0` at the end. The buildpack runs `tlmgr install` on every line in this file. So you can use single packages or these collections.
 
-When you add custom packages, keep in mind that Heroku has a maximum compressed slug-size of 300 MB (see [here](https://devcenter.heroku.com/articles/slug-compiler#slug-size)). And TeX-Live can get quite big. 
+When you add custom packages, keep in mind that Heroku has a maximum compressed slug-size of 300 MB (see [here](https://devcenter.heroku.com/articles/slug-compiler#slug-size)). And TeX-Live can get quite big.
+
+custom TeX Live version
```
Comment (codingjoe) [inline README.md]: `/custom/Custom/g`

File: README.md [hunk 2]
```diff
@@ -45,21 +45,26 @@ How does it work?
...
+----------------------
```
Comment (codingjoe) [inline README.md]: Not the same size, please add one `-`

File: README.md [hunk 3]
```diff
@@ -45,21 +45,26 @@ How does it work?
...
+By default the latest TeX Live version will be installed. You can customize the path to the installer to use by adding a file called `.texlive-path` to your project.
+It should contain only the path to the location of the TeX Live installer that you want to use without trailing `/`.
```
Comment (codingjoe) [inline README.md]: Please add an example, eg.:
```shell
echo "ftp://tug.org/historic/systems/texlive/2016/tlnet-final" > .texlive-path
```

File: bin/compile [hunk 4]
```diff
@@ -66,19 +73,20 @@ if [ ! `which pdflatex` ]; then
...
+    echo "TEXMFLOCAL $TEXLIVE_HOME/texmf-local" >> $PROF
+    echo "TEXMFSYSCONFIG $TEXLIVE_HOME/texmf-config" >> $PROF
+    echo "TEXMFSYSVAR $TEXLIVE_HOME/texmf-var" >> $PROF
+    echo "TEXMFVAR $TEXLIVE_HOME/var/texmf-var" >> $PROF
 
-    cd $TEXLIVE_HOME 
+    cd $TEXLIVE_HOME
 
-    ./install-tl --profile=$PROF
+    ./install-tl --repository=$TEXLIVE_INSTALLER_PATH --profile=$PROF
```
Comment (codingjoe) [inline bin/compile]: If the options is called "repository" maybe you should reuse this wording in the readme, and event the dotfile's name.

**Comment by anneFly** [comment]
@codingjoe thanks for the review. I changed the stuff you commented on. I also just tested the buildpack for repositories that do not have a custom url and everything worked fine (the latest latex was installed). So nothing should break for existing users.

**Comment by codingjoe** [comment]
Cool, please make sure to tag this version. I know we haven'd done that before, but we should

**Pull request merged**